### PR TITLE
Local preview script

### PR DESCRIPTION
**Pull request by zregvart**
This adds a script to help with local/uncomitted changes to documentation.

### CODE REVIEW

**Comment by simonbaird** [comment]
Possibly unrelated to this PR, but I'm getting this:

```
[11:49:42.755] FATAL (antora): Cannot use 'in' operator to search for 'worktree' in reference
```

File: hack/preview.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+cp "${ROOT_DIR}/antora-playbook.yml" "${PREVIEW_PLAYBOOK}"
+# shellcheck disable=SC2016
+yq e -i '.content.sources[].url |= sub(".*/(.*)\.git", "../$1") | .content.sources[].branches = "HEAD" | .' "${PREVIEW_PLAYBOOK}"
+
+(
+    cd "${ROOT_DIR}"
+    trap exit SIGINT
+    # shellcheck disable=SC2016,SC2046
+    while true; do
+        find $(yq '.content.sources[] | .url |= sub(".*/(.*)\.git", "../$1") | .url + "/" + .start_path' "${PREVIEW_PLAYBOOK}") | entr -s 'echo Building...; npx antora generate --clean '"${PREVIEW_PLAYBOOK}"'; echo Waiting for changes, press Ctrl+C to exit' || true
```
Comment (simonbaird) [inline hack/preview.sh]: `npx antora --stacktrace ...` to help with the plugin hacking/debugging.

**Review by simonbaird — APPROVED** [review]

**Comment by simonbaird** [comment]
The entr live updates are neat, but in practice I think I would mostly use a `hack/render.sh` that does a one-time generate.

**Comment by zregvart** [comment]
> The entr live updates are neat, but in practice I think I would mostly use a `hack/render.sh` that does a one-time generate.

It's a good idea to split into local-render and local-live, shouldn't be much work...

**Review by simonbaird — APPROVED** [review]
Nice!

**Comment by lcarva** [comment]
Thanks!

**Pull request merged**